docs/examples/src/bin/serialisation.rs

```diff
@@ -114,15 +120,17 @@ impl BootstrapServer {
     }
     // ANCHOR_END: tell_serialised
 }
+
 //ignore_control!(BootstrapServer);
 impl Provide<ControlPort> for BootstrapServer {
     fn handle(&mut self, event: ControlEvent) -> () {
         match event {
-            ControlEvent::Start => self.ctx.initialize_pool(),
+            ControlEvent::Start => self.ctx.initialise_pool(),
             _ => (), // ignore
         }
     }
 }
```

Comment:
This whole Provide impl can be deleted now, since initialisation is automatic, right?

Reply:
The Provide impl is needed for the `#[derive(ComponentDefinition)]` macro but it doesn't have to do anything, the initialisation has been removed.

Reply:
No, the commented out macro above it is sufficient.